### PR TITLE
initial static pages

### PR DESCRIPTION
**Pull request by vsauceda-cfa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,17 +11,20 @@
...
+    "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
     "mapbox-gl": "^2.1.0",
     "next": "10.0.5",
     "next-pwa": "5.0.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-mapbox-gl": "^5.1.1",
+    "styled-components": "^5.2.1",
```
Comment (timmalstead-cfa) [inline package.json:24]: is there a need for `styled-components` AND `material-ui`?

File: package.json [hunk 2]
```diff
@@ -42,12 +45,26 @@
...
+    },
+    "resolutions": {
     "**/**/object-path": "^0.11.5",
     "**/**/ini": "^1.3.6",
     "**/**/source-map-resolve": "^0.6.0",
     "**/**/chokidar": "^3.0.0",
     "twilio/axios": "^0.21.1"
-  }
-}
+    },
```
Comment (timmalstead-cfa) [inline package.json:55]: good call on below!

File: components/Burger.module.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
```
Comment (timmalstead-cfa) [inline components/Burger.module.js:1]: I'm not seeing the Burger render at all. Is there something that I am missing?

File: components/Burger.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import NextLink from 'next/link'
```
Comment (timmalstead-cfa) [inline components/Burger.tsx:1]: see comment on `Burger.module.js`

File: components/BurgerItems.module.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
```
Comment (timmalstead-cfa) [inline components/BurgerItems.module.js:1]: see comment on `Burger.module.js`

File: components/BurgerItems.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
+import NextLink from 'next/link'
```
Comment (timmalstead-cfa) [inline components/BurgerItems.tsx:1]: see comment on `Burger.module.js`

File: pages/success-stories/index.tsx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+      <Head>
+        <title>{`Santa Barbara Reentry | ${
+          language === ENGLISH ? 'Success Stories' : 'Historias de éxito'
+        }`}</title>
+      </Head>
+      {successStories.map((successStory, key) => {
+        return (
+          <div
+            key={key}
+            style={{ width: '100%', marginRight: '3rem', padding: '2rem' }}
```
Comment (timmalstead-cfa) [inline pages/success-stories/index.tsx:63]: Can we center this? Also, I think we would do well to keep a width of 90% across the project for "holder" components and pages. I used this on the tiles on the front page and think it gives it a clean and modern look. What are your thoughts?

File: pages/know-your-rights/index.tsx [hunk 8]
```diff
@@ -0,0 +1,177 @@
+import Head from 'next/head'
+import React from 'react'
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardActions from '@material-ui/core/CardActions'
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:6]: please delete unused imports.

you can run `yarn run lint-es` to help catch these kinds of problems

File: pages/know-your-rights/index.tsx [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Grid from '@material-ui/core/Grid'
+import Button from '@material-ui/core/Button'
+import Link from '@material-ui/core/Link'
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:11]: unused

File: pages/know-your-rights/index.tsx [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+import Typography from '@material-ui/core/Typography'
+import Modal from '@material-ui/core/Modal'
+import HighlightOffIcon from '@material-ui/icons/HighlightOff'
+
+const useStyles = makeStyles((theme: Theme) =>
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:16]: props unused

File: pages/know-your-rights/index.tsx [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+      color: '#13395e',
+      margin: 'auto',
+      fontSize: '3rem',
+    },
+  }),
+)
+const KnowYourRights = () => {
+  const classes = useStyles()
+  const [open, setOpen] = React.useState(false)
+  const [activeItem, setActiveItem] = React.useState({})
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:62]: state item unused

File: pages/know-your-rights/index.tsx [hunk 12]
```diff
@@ -0,0 +1,177 @@
...
+  console.log('styles')
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:63]: please delete console statements. user should not see debug info.

File: pages/know-your-rights/index.tsx [hunk 13]
```diff
@@ -0,0 +1,177 @@
...
+  const flyers = [
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:64]: less expensive to place static info outside of components

File: pages/know-your-rights/index.tsx [hunk 14]
```diff
@@ -0,0 +1,177 @@
...
+              open={open}
+              onClose={handleClose}
+              aria-labelledby="simple-modal-title"
+              aria-describedby="simple-modal-description"
+            >
+              <div className={classes.paper}>
+                <button onClick={handleClose}>
+                  <HighlightOffIcon className={classes.icons} />
+                </button>
+                {/* <img
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:162]: please delete commented text

File: pages/know-your-rights/index.tsx [hunk 15]
```diff
@@ -0,0 +1,177 @@
+import Head from 'next/head'
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:1]: General thoughts: can we center this page?
This is still pending addition of Spanish language copy, yes?

File: pages/know-your-rights/index.tsx [hunk 16]
```diff
@@ -0,0 +1,177 @@
...
+                        </Typography>
+                      </CardContent>
+                    </button>
+                  </CardActionArea>
+                </Card>
+              </Grid>
+            )
+          })}
+          <Grid item xs={12} sm={6}>
+            <Modal
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:152]: Modals not rendering documents properly
Screen Shot 2021-03-02 at 10 17 22 AM
Screen Shot 2021-03-02 at 10 17 12 AM

File: pages/know-your-rights/index.tsx [hunk 17]
```diff
@@ -0,0 +1,177 @@
...
+          {flyers.map((flyer, key) => {
+            return (
+              <Grid item xs={12} sm={3} key={key}>
+                <Card className={classes.root}>
+                  <CardActionArea>
+                    <button type="button" onClick={() => handleOpen(flyer)}>
+                      <CardMedia
+                        className={classes.media}
+                        image={flyer.profilePic}
+                        title="Contemplative Reptile"
```
Comment (timmalstead-cfa) [inline pages/know-your-rights/index.tsx:133]: Contemplative Reptile?

File: pages/about-us/index.tsx [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+        style={{ marginTop: '3rem' }}
+        gutterBottom
+        variant="h2"
+        component="h1"
+      >
+        {language === ENGLISH ? 'About Us' : 'Sobre nosotros'}
+      </Typography>
+      <Grid
+        container
+        style={{ marginTop: '10rem' }}
```
Comment (timmalstead-cfa) [inline pages/about-us/index.tsx:76]: I don't think the top margin needs to be this large

File: pages/about-us/index.tsx [hunk 19]
```diff
@@ -0,0 +1,151 @@
+import Head from 'next/head'
```
Comment (timmalstead-cfa) [inline pages/about-us/index.tsx:1]: General thoughts: same comment regarding multiple ternarys for language check as Success Stories, it's easier to do one check and then load that.

File: hooks/useOnClickOutside.tsx [hunk 20]
```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react'
```
Comment (timmalstead-cfa) [inline hooks/useOnClickOutside.tsx:1]: I'm not certain I understand how you're using this hook.

File: components/Header.tsx [hunk 21]
```diff
@@ -1,20 +1,71 @@
 import NextLink from 'next/link'
-
+import Burger from './Burger'
+import BurgerItems from './BurgerItems'
```
Comment (timmalstead-cfa) [inline components/Header.tsx:3]: Not rendering

File: components/Header.tsx [hunk 22]
```diff
@@ -1,20 +1,71 @@
...
 
 import { FreshStartLogo } from '../ui'
+const altText = {
+  english: "County of Santa Barbara Public Defender's Office Logo",
+  spanish:
+    'Logotipo de la Oficina del Defensor Público del Condado de Santa Bárbara',
+}
+
+const Header = () => {
+  const { language } = useLanguage()
```
Comment (timmalstead-cfa) [inline components/Header.tsx:17]: unused

File: components/Header.tsx [hunk 23]
```diff
@@ -1,20 +1,71 @@
...
+  const [open, setOpen] = useState(false)
+  const node = useRef()
+  useOnClickOutside(node, () => setOpen(false))
```
Comment (timmalstead-cfa) [inline components/Header.tsx:20]: how is this being used?

File: components/Header.tsx [hunk 24]
```diff
@@ -1,20 +1,71 @@
 import NextLink from 'next/link'
```
Comment (timmalstead-cfa) [inline components/Header.tsx:1]: General thoughts: the hamburger buttons are nor rendering.

Also, the links are not responsive and are not appearing at all in smaller views.
Screen Shot 2021-03-02 at 10 38 34 AM
Screen Shot 2021-03-02 at 10 38 16 AM

File: pages/checklist/index.tsx [hunk 25]
```diff
@@ -0,0 +1,676 @@
...
+import Paper from '@material-ui/core/Paper'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import { ENGLISH } from '../../types/language'
+import Grid from '@material-ui/core/Grid'
+import useLanguage from '../../hooks/useLanguage'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemAvatar from '@material-ui/core/ListItemAvatar'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction'
```
Comment (timmalstead-cfa) [inline pages/checklist/index.tsx:18]: unused

File: pages/checklist/index.tsx [hunk 26]
```diff
@@ -0,0 +1,676 @@
+import Head from 'next/head'
```
Comment (timmalstead-cfa) [inline pages/checklist/index.tsx:1]: General thoughts: can we make this smaller? it's really big for a page and is loading slowly as a result. can you write the text in the constants folder and map that?

Also, can we do this at 90% width and center by default?

File: pages/checklist/index.tsx [hunk 27]
```diff
@@ -0,0 +1,676 @@
...
+import Grid from '@material-ui/core/Grid'
+import useLanguage from '../../hooks/useLanguage'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemAvatar from '@material-ui/core/ListItemAvatar'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction'
+import ListItemText from '@material-ui/core/ListItemText'
+
+const probations = [
```
Comment (timmalstead-cfa) [inline pages/checklist/index.tsx:21]: this is pretty large, can it go in the constants folder or be called from data?

File: pages/checklist/index.tsx [hunk 28]
```diff
@@ -0,0 +1,676 @@
...
+  }),
+)
+const Checklist = () => {
+  const classes = useStyles()
+  const { language } = useLanguage()
+
+  return (
+    <div>
+      <Head>
+        <title>{`Santa Barbara Reentry | ${
```
Comment (timmalstead-cfa) [inline pages/checklist/index.tsx:109]: same comment concerning languages as success stories, i think we can slim down the language selection

**Review by timmalstead-cfa — CHANGES_REQUESTED** [review]
@vsauceda-cfa Looking good! Keep at it!

**Comment by timmalstead-cfa** [comment]
@vsauceda-cfa as a general thing, you will want to have more commits than one for a major feature change like this. Try committing each time you finish working on a file.

File: pages/success-stories/index.tsx [hunk 29]
```diff
@@ -0,0 +1,114 @@
...
+          >
+            {' '}
+            <Typography align="center" gutterBottom variant="h5" component="h2">
+              {language === ENGLISH ? 'Success Stories' : 'Historias de éxito'}{' '}
+            </Typography>
```
[the remark this answers is not in the document]
Comment (vsauceda-cfa) [reply pages/success-stories/index.tsx]: I seen that. Moved grid container outside of the .map

**Review by timmalstead-cfa — APPROVED** [review]
great work @vsauceda-cfa !

**Pull request merged**